spotbugs/src/main/java/edu/umd/cs/findbugs/detect/UncallableMethodOfAnonymousClass.java

```diff
@@ -59,6 +59,11 @@ public UncallableMethodOfAnonymousClass(BugReporter bugReporter) {
 
     @Override
     public void visitJavaClass(JavaClass obj) {
+        if (Values.DOTTED_JAVA_LANG_ENUM.equals(obj.getSuperclassName()) ||
+                Values.DOTTED_JAVA_LANG_RECORD.equals(obj.getSuperclassName())) {
+            return;
```

Comment:
There could be extra methods added to an Enum that would validly benefit from the rule, though.

Reply:
Hmm, you are right. I will change this to only disregard auto-generated methods.